.github/workflows/commit-lint.yaml

```
@@ -42,13 +42,17 @@ jobs:
+            npx commitlint --from $BASE_SHA --to ${{ github.event.pull_request.head.sha }} --verbose
           else
+            # For direct pushes, only check new commits
             npx commitlint --from ${{ github.event.before }} --to ${{ github.event.after }} --verbose
           fi
         
       - name: Check PR title
         if: github.event_name == 'pull_request' || github.event_name == 'pull_request_target'
         working-directory: notes
         run: |
-          echo "${{ github.event.pull_request.title }}" | npx commitlint --verbose
+          echo "${{ github.event.pull_request.title }}" | npx commitlint --verbose
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)